soapui/src/main/java/com/eviware/soapui/impl/support/HttpUtils.java

```diff
@@ -53,7 +53,7 @@ public static String completeUrlWithHttpIfProtocolIsNotHttpOrHttpsOrPropertyExpa
         if (StringUtils.isNullOrEmpty(endpoint)) {
             return endpoint;
         }
-        endpoint = endpoint.toLowerCase().trim();
+        endpoint = endpoint.trim();

```

Comment:
Hi Ole,
Its an honour to hear from you! :-)
I sent my signed CLA to Manne - shall I send it to him again? I don't have your email address.
You are of course right about the possibility of someone entering HTTP - I'll add a test & fix for that - thanks for spotting that, good catch!
Thanks for your help,
Rupert